dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -248,6 +248,7 @@ private boolean resolveFullDependencies() {
    * @throws DependencyResolutionException when there is a problem in resolving dependency. This
    *     happens only when graphTraversalOption is FULL_DEPENDENCY or FULL_DEPENDENCY_WITH_PROVIDED.
    */
+  @SuppressWarnings("unchecked")
```

Comment:
What's unchecked? Please add a comment explaining why it's safe.

Is there a smaller scope in which you can apply this?

Reply:
It's clone. Done.